portfolio/src/main/java/com/google/sps/libraries/VerifyUserCommentID.java

```diff
@@ -0,0 +1,46 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.verifyusercommentid;
+
+import com.google.appengine.api.datastore.*;
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import java.io.IOException;
+import java.io.PrintWriter;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet responsible to remove every comment requested by the user */
+@WebServlet("/verify-user-comment-id")
```

Comment:
```suggestion
@WebServlet("/remove-comment")
```
I think the name of the servlet is a bit confusing. The comment above says this servlet is used to remove a comment, so the name should reflect that. Verification is an implementation detail which you don't need to include in the servlet name.

Reply:
Yes, my bad! Actually that servlet is just to compare both comment and users id and send if the user is able to delete the comment or not. Going to change the class comment right now!